linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -734,3 +736,162 @@ impl LocalNet {
         Ok(())
     }
 }
+
+//
+// The Shared stuff
+//
+
+/// The number of simultaneous sets of validators
+#[cfg(with_testing)]
+const N_SIMULTANEOUS_RUNS: usize = 5;
```

Comment:
I don't understand this.

Reply:
This is a limitation on the number of simultaneous runs being executed. If set to 1, then the tests are run serially as in the code before this CI. If set to a very large value, then all tests are run in parallel.